pkg/apis/flyteworkflow/v1alpha1/iface.go

```diff
@@ -153,6 +153,25 @@ func (b BranchNodePhase) String() string {
 	return "Undefined"
 }
 
+//go:generate enumer --type WorkflowOnFailurePolicy --json --yaml --trimprefix=WorkflowOnFailurePolicy
+
+// Failure Handling Policy
+type WorkflowOnFailurePolicy int32
```

Comment:
Interesting, so if we add one more element, we will need to add it here?

Reply:
I think I like this, but it does not follow our other enums right?

Reply:
Is this better?